sdk/storage/azure-storage-blob/setup.py

```diff
@@ -69,7 +69,7 @@
     license='MIT License',
     author='Microsoft Corporation',
     author_email='ascl@microsoft.com',
-    url='https://github.com/Azure/azure-storage-python',
+    url='https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-blob',
```

Comment:
Not sure if we want to keep use https://github.com/Azure/azure-sdk-for-python or the detailed one. Any thoughts?

Reply:
We might want to use the main link..that's what other packages are doing.

Reply:
I just took a look at azure-identity package and they are using the detailed one https://pypi.org/project/azure-identity/
so probably let's use the detailed one for all?

Reply:
hmm, sounds good..in that case, lets change for queues and files as well :)

Reply:
we have already did I think?